server/axios_api.js

```diff
@@ -1,61 +1,31 @@
-var axios = require("axios").default;
-var HOST = "movies-tvshows-data-imdb.p.rapidapi.com";
-var API_KEY = "030611f594msh73349b7b00af6e8p16d57ejsncbf1b45a6165";
-var URL = "https://movies-tvshows-data-imdb.p.rapidapi.com/";
-
-
-
-function get_details(id) {
-  var options = {
-    method: 'GET',
-    url: URL,
-    params: {type: 'get-movie-details', imdb: id},
-    headers: {
-      'x-rapidapi-host': HOST,
-      'x-rapidapi-key': API_KEY
-    }
-  };
-  axios.request(options).then(function (response) {
-    console.log("rating:"+response.data["imdb_rating"]+"\n"+"popularity:"+response.data["popularity"]+"\n"+"vote:"+response.data["vote_count"]);
-  }).catch(function (error) {
-    console.error(error);
-  });
-
+const { default: axios } = require("axios");
+const RATINGS = {
+    RottenTomatoes: "",
+    IMDB: ""
 }
 
-
-function get_id(t){
-  var options = {
-    method: 'GET',
-    url: URL,
-    params: {type: 'get-movies-by-title', title: t},
-    headers: {
-      'x-rapidapi-host': HOST,
-      'x-rapidapi-key': API_KEY
+const getDetails = async(title) => {
+    var API_KEY="631772a5";
+    var URL=`https://www.omdbapi.com/?apikey=${API_KEY}&t=${title}`;
+
+    try {
+         const response = await axios.get(URL)
+         const ratings = response.data['Ratings'];
+         ratings.forEach(element => {
+             if (element['Source']=="Internet Movie Database"){
+                RATINGS["RottenTomatoes"]=element["Value"];
```

Comment:
we can access obj properties using dot notation?

const obj = {
firstName: "Isha",
lastName: "gupta"
}

console.log(obj.firstName) instead of console.log(obj["firstName"])

Though both will print same results, object notation is more standard form I think